.github/workflows/_test_build_nix.yaml

```diff
@@ -0,0 +1,32 @@
+name: Test Build *Nix
+
+on:
+  workflow_call:
+    secrets:
+      docker_hub_id:
+        description: "docker hub id"
+        required: true
+      docker_hub_password:
+        description: "docker hub password"
+        required: true
```

Comment:
are these really needed ? @sigilioso  removed from their action, i think this is legacy

Reply:
I think that without we could have issues with the rate limits of Docker 🤔 
I would keep it for the pipelines executing often like the push one (if the rate-limit is still a thing)

Reply:
Good point. We can talk about it during the standup, but it is indeed used much more than the release workflow 🤔

Reply:
As discussed, we remove them for now.